templates/BodyParamSnippet.stencil

```diff
@@ -1,3 +1,3 @@
-{% if op.params.body.strategy == "plain" or op.params.body.strategy == "unixTime" or op.params.body.strategy == "plainNullable" %}
+{% if op.params.body.strategy == "plain" or op.params.body.strategy == "unixTime" or op.params.body.strategy == "plainNullable" or op.params.body.strategy == "byteArray" %}
```

Comment:
I tried changing this to `{% if op.params.body.strategy != “flattened” %}` but stencil file does not work.

Reply:
You could always do:
```swift
{% if op params.body.strategy == "flattened" %}{% else %}
...
```

Reply:
Also, from here (https://stencil.fuller.li/en/latest/builtins.html#built-in-tags) it looks like this might work:
```swift
{% if not op.params.body.strategy == "flattened" %}
```

I'm not sure why the != operator isn't working, since the documentation says it should. I'd recommend posting an issue in the stencil repo, since that would be a bug.

Reply:
Thanks for the suggestion. I find the cause of the issue. 
We should check if op.params.body is not nil in templates/OperationSnippet.stencil before include templates/BodyParamSnippet.stencil. Otherwise, we will have op.params.body.strategy equals to ""